### PR TITLE
[CU-860pajcw7] Teamhub - Add Title + Description Sections to Form Editor

### PR DESCRIPTION
**Pull request by andaniel01**

### CODE REVIEW

File: pages/form/edit/[formName].js [hunk 1]
```diff
@@ -483,6 +487,38 @@ const FormEditor = () => {
                         },
                     ]}
                 />
+
+                <Card 
+                    display='grid'
+                    gridTemplateColumns='1fr'
+                    gridColumnGap={theme.space[5]}
```
Comment (SteveShengStar) [inline pages/form/edit/[formName].js:494]: I don't think grid-column-gap is necessary.

**Review by SteveShengStar — COMMENTED** [review]
Looks good, just left 1 comment.

Just curious, does the new title/description you fill in get saved to the back end ?

Also, when you come back to the page (or refresh it), does the new title/description you set get populated in the new fields you created ?

File: pages/form/edit/[formName].js (hunk 1 above)
Comment (andaniel01) [reply pages/form/edit/[formName].js:494]: you're right it's not necessary, will remove it.

**Comment by andaniel01** [comment]
> Looks good, just left 1 comment.
> 
> Just curious, does the new title/description you fill in get saved to the back end ?
> 
> Also, when you come back to the page (or refresh it), does the new title/description you set get populated in the new fields you created ?

@SteveShengStar The new title/description does get saved in the backend, when refreshing the webpage. I also checked the forms documents in mongoDB and the changes are reflected there too.

**Review by SteveShengStar — APPROVED** [review]

**Pull request merged**